tests/python/relax/test_tvmscript_parser.py

```diff
@@ -1309,6 +1309,9 @@ def add(
 
         @R.function
         def main(x: R.Tensor((2, 4), dtype="float32")) -> R.Tensor((10,), dtype="float32"):
+            # slight hack: normally, we would prefer to use True, but the func attrs, when printed,
+            # will have it as 1, so it would fail roundtripping otherwise
+            R.func_attr({"ForcePure": 1})
```

Comment:
@yongwww, would you happen to know how I could avoid this (print the attribute as `True` instead of 1)? It's not super-pressing, but it would help for roundtripping

More important: Would you be able to advise how we could have a special case in the pretty-printer for `call_pure`? I would like to show it properly wrapping the "inner call"

Reply:
it related printing should be https://github.com/apache/tvm/blob/unity/src/script/printer/relax/function.cc#L54-L58. Feel free to leave a todo for me